pkg/controller.v2/controller.go

```diff
@@ -377,6 +398,48 @@ func (tc *TFJobController) syncTFJob(key string) (bool, error) {
 	return true, err
 }
 
+// SyncPdb will create a PDB for gang scheduling by kube-arbitrator.
+func (tc *TFJobController) syncPdb(tfjob *tfv1alpha2.TFJob) (*v1beta1.PodDisruptionBudget, error) {
+	// Sum of tfjob's replicas
+	tfjobReplicas := int32(0)
+	for _, r := range tfjob.Spec.TFReplicaSpecs {
+		tfjobReplicas += *r.Replicas
+	}
+
+	// Non-distributed training is not required gang scheduling
+	if tfjobReplicas < 2 {
+		return nil, nil
+	}
+
+	// Check the pdb exist or not
+	pdbName := "tf-job-pdb-" + tfjob.Name
+	pdb, err := tc.kubeClientSet.PolicyV1beta1().PodDisruptionBudgets(tfjob.Namespace).Get(pdbName, metav1.GetOptions{})
+	if err == nil || !k8serrors.IsNotFound(err) {
+		log.Warnf("PDB: %v already exists or %v", pdbName, err)
+		if err == nil {
+			err = errors.New(string(metav1.StatusReasonAlreadyExists))
+		}
+		return pdb, err
+	}
+
+	// Create pdb for gang scheduling by kube-arbitrator
+	minAvailable := intstr.FromInt(int(tfjobReplicas))
+	createPdb := &v1beta1.PodDisruptionBudget{
```

Comment:
@jinzhejz

Does the code work for you?

Reply:
Can we set `OwnerReference` with controller? In kube-arbitrator, we prefer to use `OwnerReference` to avoid `Selector` because of performance concern.

Reply:
BTW, when is PDB deleted?

Reply:
Yeah, we are going to add owner reference now.

Reply:
Do you mean to set the owner reference for the pdb?

Reply:
We want to rely on Kubernetes garbage collector for the pdb currently. While we should implement the logic about deleting pdb when deleting the tfjob.

Reply:
> Do you mean to set the owner reference for the pdb?

Yes, set OwernReference for the PDB, and also set `OwerReference.Controller` to `true`

> We want to rely on Kubernetes garbage collector for the pdb currently.

That's OK to me :)

Reply:
@codeflitting 

We come to an agreement, please update the code, thanks for your contribution!